rs/src/main.rs

```diff
@@ -0,0 +1,58 @@
+mod version_mod;
+
+use anyhow::{Context, Result};
+use clap::{Parser, Subcommand};
+use std::fs;
+
+/// Simple CLI for Artisan Tools.
+#[derive(Parser)]
+#[command(name = "artisan-tools")]
+#[command(about = "Artisan Tools CLI", long_about = None)]
```

Comment:
Not sure why we set `long_about = None`.

However we need to add version to enable `--version` and `-V` to automagically resolve to the version of the crate

```suggestion
#[command(about = "Artisan Tools CLI", version)]
```

Reply:
That's to make it clear to everyone that we don't really know what Artisan Tools is 🤣